src/secret-storage.ts

```diff
@@ -200,7 +202,9 @@ export interface SecretStorageCallbacks {
     ) => Promise<[string, Uint8Array] | null>;
 }
 
-interface SecretInfo {
+export type SecretInfoKey = Assignable<AccountDataEvents, SecretInfo>;
```

Comment:
Also, I feel like this should probably be called `SecretStorageKey` or something that makes it more obviously related to, y'know, secret storage.